build.bat

```diff
@@ -16,11 +16,12 @@ IF "%VisualStudioVersion:~0,2%" == "16" GOTO :START_BUILD
 set GENERATOR="Visual Studio 17 2022"
 
 :START_BUILD
-set cmake_exe="%VSINSTALLDIR%Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe"
+REM Add path to VS cmake as the fallback.
+set PATH=%PATH%;"%VSINSTALLDIR%Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe"
```

Comment:
This and the change to setup.py work in the CIs so that the cmake version that is explicitly installed and added to the path is preferred.

If we want to hardcode defaulting to the VS cmake we can't update the required cmake version.

Reply:
Using a explicit cmake is intentional since cmake installed by pip are not stable enough which I encountered before.  Also,  this change may bring some uncertainty of which cmake was used to build. it will bring some difficulty on checking some CI build issues.
To override cmake, we may use some environment variable to do that. 
Did you encounter any case that VS/cmake has to be replaced? VS/cmake actually keeps updating and it looks like a LTS version for Windows platform.

Reply:
Yeah - the win32.wheel build breaks as the VS it's using only has cmake 3.20. The CI itself is has more a recent cmake installed.

Reply:
I checked VS 2022 installation, it is 3.25.1, 
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin>cmake --version
cmake version 3.25.1-msvc1

is 3.20 due to VS 2019?

If it seeks in general %PATH% for cmake, it actually uses cmake from pip package because of cmake dependency defined in pyproject.toml. And this cmake edition is not good for official build.

Reply:
> 2023-05-15T04:43:09.6713775Z   "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -S C:\a\_work\1\s -B .scb\temp.win-amd64-cpython-37\Release -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=C:\a\_work\1\s\.scb\lib.win-amd64-cpython-37\onnxruntime_extensions -DOCOS_BUILD_PYTHON=ON -DOCOS_EXTENTION_NAME=_extensions_pydll.cp37-win_amd64.pyd -DCMAKE_BUILD_TYPE=Release
2023-05-15T04:43:09.6714176Z   -- Building for: Visual Studio 16 2019
2023-05-15T04:43:09.6714372Z   CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
2023-05-15T04:43:09.6714570Z     CMake 3.25 or higher is required.  You are running version
2023-05-15T04:43:09.6714735Z     3.20.21032501-MSVC_2

It's using VS2019. Did you want to change that so it uses VS 2022?

Reply:
Yes, please.
In get_vsdevcmd.ps1, I selected the latest VS installation if there is multiple VS versions installed.